packages/xrpl/src/models/methods/serverInfo.ts

```diff
@@ -136,6 +136,10 @@ export interface ServerInfoResponse extends BaseResponse {
        * overall network's load factor.
        */
       load_factor?: number
+      /**
+       * The network id of this server
+       */
+      network_id?: number
```

Comment:
Why are all the network ID changes in here as well?

Reply:
This was added in because the hooks amendment includes the network id functionality.

Reply:
We already have a separate PR for that (#2216), it shouldn't also be in this PR.

Reply:
It becomes tricky because network_id is required for hooks testnet.  #2216 will need to be merged first.  Also the network_id code will not be hidden behind an amendment.